internal/controllers/user_devices_controller.go

```diff
@@ -1919,6 +1976,26 @@ type AdminRegisterUserDevice struct {
 	Verified    bool    `json:"verified"`
 }
 
+type GetUserDevicesWithCredentialsRequest struct {
+	AuthorizationCode string `json:"authorizationCode"`
+	RedirectURI       string `json:"redirectUri"`
+	Region            string `json:"region"`
+}
+
+type GetUserDevicesWithCredentialsResponseWrapper struct {
+	Vehicles []GetUserDevicesWithCredentialsResponse
+}
+
+type GetUserDevicesWithCredentialsResponse struct {
+	ID                 int    `json:"id"`
+	VehicleID          int    `json:"vehicleId"`
+	VIN                string `json:"vin"`
+	Make               string
+	Model              string
+	Year               int
+	DeviceDefinitionID string
```

Comment:
Are we populating these?

We should probably nest the device definition stuff, so like

```
{
  vehicles: [
    {id: "5434637", vin: "5YJ...", definition: {id: "2fU8988...", make: "Toyota", ....}},
   ...
  ]
}